src/ngx_http_lua_worker.c

```diff
@@ -15,12 +15,14 @@
 
 static int ngx_http_lua_ngx_worker_exiting(lua_State *L);
 static int ngx_http_lua_ngx_worker_pid(lua_State *L);
+static int ngx_http_lua_ngx_worker_id(lua_State *L);
+static int ngx_http_lua_ngx_worker_count(lua_State *L);
 
 
 void
 ngx_http_lua_inject_worker_api(lua_State *L)
 {
-    lua_createtable(L, 0 /* narr */, 2 /* nrec */);    /* ngx.worker. */
+    lua_createtable(L, 0 /* narr */, 4 /* nrec */);    /* ngx.worker. */
```

Comment:
I think you also need to update the corresponding tests in `t/062-count.t` to reflect this change.